### PR TITLE
[WIP] sdl_net: homepage change, style changes (brew audit --strict)

### PR DESCRIPTION
**Pull request by L2G**
Code reviewers, please look specifically at the test block I added. Since this is a library, there is no binary _per sé_, so doing a validation with `pkg-config` seemed like the logical thing to do. Maybe there is a better kind of test to do? Or maybe there is a more idiomatic way to check the output of a command in a test block?

### CODE REVIEW

File: Library/Formula/sdl_net.rb [hunk 1]
```diff
@@ -1,20 +1,18 @@
-require 'formula'
-
 class SdlNet < Formula
-  homepage 'http://www.libsdl.org/projects/SDL_net/'
-  url 'http://www.libsdl.org/projects/SDL_net/release/SDL_net-1.2.8.tar.gz'
-  sha1 'fd393059fef8d9925dc20662baa3b25e02b8405d'
+  homepage "https://www.libsdl.org/projects/SDL_net/release-1.2.html"
+  url "http://www.libsdl.org/projects/SDL_net/release/SDL_net-1.2.8.tar.gz"
```
Comment (tdsmith) [inline Library/Formula/sdl_net.rb]: if the homepage is SSL'd, is this SSL'd?

**Comment by tdsmith** [comment]
In general the best way to test a library is to build example code against it.

File: Library/Formula/sdl_net.rb (hunk 1 above)
Comment (L2G) [inline Library/Formula/sdl_net.rb]: Um… good point. :)

**Comment by DomT4** [comment]
This needs rebasing against master.

**Comment by bfontaine** [comment]
See [`tinyxml2.rb`](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/tinyxml2.rb#L22) for an example of a test for a library formula.

**Comment by bfontaine** [comment]
I don’t see the test in the diff, did you remove it?

**Comment by L2G** [comment]
A test has not yet been written.

**Comment by bfontaine** [comment]
Oh ok, I thought you wrote it before.

**Comment by dunn** [comment]
Afraid this needs rebasing again.

**Comment by DomT4** [comment]
Fixed the remaining two tweaks in https://github.com/Homebrew/homebrew/commit/af1df61f6bc90787c83af736e39e16103f2774de.

If you want to follow in with a test, that'd be awesome. Thanks for the submission.

**Pull request closed without merging**